src/theme/index.ts

```diff
@@ -153,3 +153,15 @@ export const PATheme: ThemeSpec = {
     resourceScheme: "pa",
     componentScheme: "pa",
 };
+
+export const A11yTheme: ThemeSpec = {
```

Comment:
we might want to keep the naming consistent - is the product called A11y Playground with a A11yPlaygroundTheme and `a11yPlayground` schemes, or A11y with a `A11yTheme`?